beamer/tests/agent/unit/test_relayer.py

```diff
@@ -0,0 +1,47 @@
+from pathlib import Path
+from subprocess import CalledProcessError
+from unittest.mock import patch
+
+from hexbytes import HexBytes
+
+from beamer.agent.relayer import run_relayer_for_tx
+from beamer.tests.agent.unit.utils import make_address, make_context
+
+
+@patch("beamer.agent.relayer.subprocess.run")
+@patch("beamer.agent.relayer.get_relayer_executable")
```

Comment:
I don't like the fact that we use this much mocking. We need to try to avoid messing with the implementation so much.

My thinking is, we should be able to get the same effect (check that the password was redacted) by e.g. providing a path to a nonexistent keyfile, or giving a wrong password etc. In both cases the relayer should exit with an error and we should be able to get the `CalledProcessError` and check for the password. And we can avoid patching our functions.